Formula/papilo.rb

```diff
@@ -1,8 +1,8 @@
 class Papilo < Formula
   desc "Parallel Presolve for Integer and Linear Optimization"
   homepage "https://github.com/scipopt/papilo"
-  url "https://github.com/scipopt/papilo/archive/refs/tags/v2.1.2.tar.gz"
-  sha256 "7e3d829c957767028db50b5c5085601449b00671e7efc2d5eb0701a6903d102f"
+  url "https://scipopt.org/download/release/scipoptsuite-8.0.3.tgz"
```

Comment:
artifact seems totally different?

Reply:
Yes. I updated it now.

Reply:
I still don't quite understand, can you elaborate?

Reply:
Any comments or thoughts?